Tests/BuildTests/BuildPlanTests.swift

```diff
@@ -1151,17 +1171,41 @@ final class BuildPlanTests: XCTestCase {
       #if os(macOS)
         XCTAssertMatch(aTarget, ["-target", "x86_64-apple-macosx10.13", .anySequence])
       #else
-        XCTAssertMatch(aTarget, ["-target", "x86_64-unknown-linux", .anySequence])
+        #if arch(x86_64)
```

Comment:
Unnecessarily duplicated; just use the `defaultTargetTriple` variable.

Reply:
@jakepetroules  if using the variable, but got compile errors like: 

      /home/work/sw/swift5/swiftpm/Tests/BuildTests/BuildPlanTests.swift:1174:34: error: type of   expression is ambiguous without more context
         XCTAssertMatch(aTarget, ["-target", PLATFORM, .anySequence] )
                                 ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      --- bootstrap: error: build failed with exit status 1

Tried to find a way to make it work, but failed.  Do you have idea to solve it?
Thanks

Reply:
Try:

    XCTAssertMatch(aTarget, [.equal("-target"), .equal(defaultTargetTriple), .anySequence])